x-pack/plugins/canvas/canvas_plugin_src/renderers/advanced_filter/index.tsx

```diff
@@ -4,19 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import ReactDOM from 'react-dom';
 import React from 'react';
+import ReactDOM from 'react-dom';
+import { RendererFactory } from '../types';
 import { AdvancedFilter } from './component';
 
-export const advancedFilter = () => ({
+export const advancedFilter: RendererFactory = () => ({
   name: 'advanced_filter',
   displayName: 'Advanced filter',
   help: 'Render a Canvas filter expression',
   reuseDomNode: true,
   height: 50,
-  render(domNode, config, handlers) {
+  render(domNode, _, handlers) {
     ReactDOM.render(
-      <AdvancedFilter commit={handlers.setFilter} filter={handlers.getFilter()} />,
+      <AdvancedFilter commit={handlers.setFilter} value={handlers.getFilter()} />,
```

Comment:
This breaks the advanced_filter component, it never renders a value anymore.

The container still expected that the argument is `filter`, not `value`. See [the container code](https://github.com/elastic/kibana/blob/master/x-pack/plugins/canvas/canvas_plugin_src/renderers/advanced_filter/component/index.ts#L16).

Reply:
Fixed.